src/cityreader/cityreader.py

```diff
@@ -14,14 +14,45 @@
 #
 # Note that the first line of the CSV is header that describes the fields--this
 # should not be loaded into a City object.
+import csv
+
+
+class City:
+    def __init__(self, name, lat, lon):
+        self.name = name
+        self.lat = lat
+        self.lon = lon
+
+    def __repr__(self):
+        return f"{self.name}, {self.lat}, {self.lon}"
+
+
 cities = []
 
+# I don't know if this was intentional, but I had to write another function to remove trailing zeros in order to pass the test file.
```

Comment:
Yes, I believe this is just a little gotcha that they dropped in. Prior to adjusting the output, you'll get `lat` and `lon` as strings, while the test file expects floats. Good job finding a workaround. You could even call float directly on those values and it will trim off the trailing zeroes for you. Great instincts creating a helper function 🔥